docs/aws/karpenter.cloudformation.yaml

```diff
@@ -1,64 +1,10 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: Resources used by https://github.com/awslabs/karpenter
 Parameters:
-  OpenIDConnectIdentityProvider:
-    Type: String
-    Description: "Example oidc.eks.us-west-2.amazonaws.com/id/1234567890"
   ClusterName:
     Type: String
     Description: "EKS cluster name"
 Resources:
-  KarpenterControllerRole:
```

Comment:
Now users are forced to install karpenter using eksctl (to create this role). Thoughts on alternatives?

Reply:
For the example we're only showing eksctl (and how easy it is) We should absolutely document how to do this manually and what the requirements are (as well how to customize it). That all seemed like out of scope for the getting started guide.

Reply:
SGTM